spec/features/embedded_apps_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-RSpec.feature "Embedded apps", type: :feature do
+RSpec.feature "Embedded apps" do
```

Comment:
Preco si tu zmazal `type: :feature`? Ma to daky dovod?

Reply:
Nedeklarujeme to uz vlastne tu '_RSpec.**feature** "Embedded apps", type: :feature do_' , my to pride ako duplikacia.

feature test je vlastne syntaxne
_RSpec.describe "Embedded apps", type: :feature do_
aj
 _RSpec.feature "Embedded apps" do_
to iste ci?